### PR TITLE
Add Irssi, Vmail and QuteBrowser

### PR DESCRIPTION
**Pull request by sajattack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ Two Chinese version of this list are available [here](https://github.com/alim0x/
 - [GhettoSkype](https://github.com/stanfieldr/ghetto-skype) - open source web wrapper for Skype. [![Open-Source Software][OSS Icon]](https://github.com/stanfieldr/ghetto-skype) ![Freeware][Freeware Icon]
 - [Gitter](https://gitter.im/) - Gitter — Where developers come to talk. Gitter is designed to make community messaging, collaboration and discovery as smooth and simple as possible. [![Open-Source Software][OSS Icon]](https://github.com/gitterHQ/services) ![Freeware][Freeware Icon]
 - [HexChat](https://hexchat.github.io/) - HexChat is an IRC client based on XChat, but unlike XChat it’s completely free for both Windows and Unix-like system.[![Open-Source Software][OSS Icon]](https://github.com/hexchat) ![Freeware][Freeware Icon]
+- [Irssi](https://github.com/irssi/irssi) - The client of the future. A terminal-based IRC client. [![Open-Source Software][OSS Icon]] ![Freeware][Freeware Icon]
```
Comment (LewisVo) [inline README.md]: I think you should replace the decription. This description is too general and brings no value, especially the term "The client of the future". I see on their project page the default description goes like this : " Irssi is a modular chat client that is most commonly known for its text mode user interface ". This one is better. Moreover, they list a few features, which would deserves mentioning in the description 👍

File: README.md [hunk 2]
```diff
@@ -855,7 +855,8 @@ firewall zones to define the trust level of network connections or interfaces. !
  34. [alim0x](https://github.com/alim0x)
  35. [Spacebody](https://github.com/Spacebody)
  36. [AyoIsaiah](https://github.com/ayoisaiah)
- 37. *Everyone @ reddit.com/r/linux & reddit.com/r/ubuntu & vozforums.com for many suggestions and compliments*
+ 37. [sajattack](https://github.com/sajattack)
```
Comment (LewisVo) [inline README.md]: Haha this is fine. I am gonna replace all the list with a link to https://github.com/LewisVo/Awesome-Linux-Software/graphs/contributors. Right now there are about 50 contributors, which makes the list no longer suitable. ;)

File: README.md [hunk 3]
```diff
@@ -307,6 +308,7 @@ Two Chinese version of this list are available [here](https://github.com/alim0x/
 - [Thunderbird](https://www.mozilla.org/en-US/thunderbird/) - Thunderbird is a free email application that’s easy to set up and customize and it’s loaded with great features. ![Freeware][Freeware Icon]
 - [Trojita](http://trojita.flaska.net/download.html) - A super fast desktop email client for Linux.[![Open-Source Software][OSS Icon]](https://gerrit.vesnicky.cesnet.cz/r/#/dashboard/?)![Freeware][Freeware Icon] title=Trojit%C3%A1&Pending=project:trojita+is:open&Recently+Closed=project:trojita+is:closed
 - [Wmail](https://github.com/Thomas101/wmail) - Gmail & Google Inbox unofficial desktop client for linux. [![Open-Source Software][OSS Icon]](https://github.com/Thomas101/wmail)![Freeware][Freeware Icon]
+- [Vmail](http://danielchoi.com/software/vmail.html) - Vim-like Gmail client [![Open-Source Software][OSS Icon]](https://github.com/danchoi/vmail)![Freeware][Freeware Icon]
```
Comment (LewisVo) [inline README.md]: Vmail should be over Wmail ( alphabetical order)

File: README.md [hunk 4]
```diff
@@ -410,6 +412,7 @@ Two Chinese version of this list are available [here](https://github.com/alim0x/
 - [Min](https://minbrowser.github.io/min) - A smarter, faster web browser.![Freeware][Freeware Icon] [![Open-Source Software][OSS Icon]](https://github.com/minbrowser/min)
 - [Opera](http://www.opera.com/) - Opera browser is everything you need to do more on the web. ![Freeware][Freeware Icon]
 - [Popcorntime](https://www.popcorn-time.to/) - Watch torrent movies instantly. ![Freeware][Freeware Icon] [![Open-Source Software][OSS Icon]](https://www.popcorn-time.to/source.html)
+- [QtBrowser](https://www.qutebrowser.org/) - A vim-like web browser. ![Freeware][Freeware Icon] [![Open-Source Software][OSS Icon]](https://github.com/qutebrowser/qutebrowser)
```
Comment (LewisVo) [inline README.md]: Web browser should not be under this category, but the "internet" category.

File: README.md (hunk 1 above)
Comment (sajattack) [inline README.md]: I actually got "The client of the future" off their github page, but you're right, that one sounds better.

**Comment by LewisVo** [comment]
Hi @sajattack , thank you for your contribution. I have tried to run all your suggestions and they are all working well. There are just some minor errors that I noted. I am awating your reponses. 😄

**Comment by LewisVo** [comment]
This looks very well-written now. Thank you @sajattack . Have a nice day ;)

**Comment by sajattack** [comment]
Thank you as well.

**Comment by sajattack** [comment]
I just noticed I accidentally put extra brackets around the OSS icon for Irssi.

**Comment by LewisVo** [comment]
@sajattack , dont worry, I'll fix that later. ;)

**Comment by sajattack** [comment]
Ok great! :)

**Pull request merged**